### PR TITLE
Amendments to Safari cheatsheet

### LINKED ISSUE #1811: Safari Cheat Sheet: Add tab switching
Please add the following shortcuts:

Control-tab: Switch tabs (forward)
Shift-control-tab Switch tabs (reverse)

/cc @ankushg07

### PR DESCRIPTION
**Pull request by altern8tif**
Fixes #1811.

Addresses PR #1809 as well.

------
IA Page:  http://duck.co/ia/view/meta_cpan

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/safari.json [hunk 1]
```diff
@@ -13,8 +13,11 @@
               "key":"[⌘] [T]",
               "val":"Open new tab"
            },{
-              "key":"[⌘] [←] / [→]",
-              "val":"Jump to next / jump to previous tab"
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/safari.json]: Not sure if we should remove this as it's still a valid shortcut.

File: share/goodie/cheat_sheets/json/safari.json [hunk 2]
```diff
@@ -13,8 +13,11 @@
...
+              "key":"[Ctrl] [Tab]",
+              "val":"Switch to next tab"
+           },{
+              "key":"[Ctrl] [Shift [Tab]",
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/safari.json]: should be `[Ctrl] [Shift] [Tab]`

File: share/goodie/cheat_sheets/json/safari.json (hunk 1 above)
Comment (altern8tif) [inline share/goodie/cheat_sheets/json/safari.json]: This actually doesn't work for me on Safari (El Capitan).

File: share/goodie/cheat_sheets/json/safari.json (hunk 2 above)
Comment (altern8tif) [inline share/goodie/cheat_sheets/json/safari.json]: Ah! Thanks for spotting...

File: share/goodie/cheat_sheets/json/safari.json (hunk 1 above)
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/safari.json]: Ahh ok, looks like the original source for this sheet is for Safari 6/7. Might want to update the source to https://support.apple.com/kb/PH21483 (Safari 9), check all the shortcuts are still valid and add any additional ones :+1:

**Comment by moollaza** [comment]
@altern8tif Thanks! This LGTM :+1:

**Pull request merged**